src/Microsoft.Health.Dicom.SqlServer.UnitTests/Features/ExtendedQueryTag/AddInstanceTableValuedParametersBuilderTests.cs

```diff
@@ -23,7 +24,7 @@ public void GivenSupportedDicomElement_WhenRead_ThenShouldReturnExpectedValue(Di
             DicomDataset dataset = new DicomDataset();
             dataset.Add(element);
             QueryTag tag = new QueryTag(element.Tag.BuildExtendedQueryTagStoreEntry(vr: element.ValueRepresentation.Code));
-            var parameters = ExtendedQueryTagDataRowsBuilder.Build(dataset, new QueryTag[] { tag });
+            var parameters = ExtendedQueryTagDataRowsBuilder.Build(dataset, new QueryTag[] { tag }, SchemaVersion.V5);
```

Comment:
Since there's logic based this value, how about including schema version as an argument to this test method and specifying it for each data member?

Reply:
Yep, that would be valuable! I will make the necessary changes here.

Reply:
Latest iteration has the updated tests that run for all the schema versions between min and max.